src/fidesops/schemas/user_permission.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+from pydantic import validator
+from fastapi import HTTPException
+from starlette.status import HTTP_422_UNPROCESSABLE_ENTITY
+from fidesops.schemas.base_class import BaseSchema
+from fidesops.api.v1.scope_registry import SCOPE_REGISTRY
+
+
+class UserPermissionsCreate(BaseSchema):
+    """Data required to create a FidesopsUserPermissions record"""
+
+    scopes: List[str]
+
+    @validator("scopes")
+    def validate_scopes(cls, scopes: List[str]) -> List[str]:
+        """Validates that all incoming scopes are valid"""
+        if not all([scope in SCOPE_REGISTRY for scope in scopes]):
```

Comment:
I'd preference using set arithmetic here. Something like:
```
diff = set(SCOPE_REGISTRY).difference(set(scopes))
```
that way we can include the invalid scopes (the elements of `diff`) as part of the error message.

Reply:
Good idea. I'll add some tests to make sure the error message contains the invalid scopes.